autocomplete_mention/main.js

```diff
@@ -0,0 +1,89 @@
+/*global $:false */
+
+// needs jQuery, jQuery-ui, jQuery-ui smoothness CSS theme
```

Comment:
Where are you adding these dependencies? Is there another autocomplete module that is more flexable? typeahead.js? awesomplete? horsey?